docs/docs/event-trackers/amplitude.mdx

```diff
@@ -6,7 +6,7 @@ slug: amplitude
 ---
 
 import DefaultInstructions from './_default-event-instructions.mdx'
-const { title, description, sidebar_label, slug } = frontMatter
+export const { title, description, sidebar_label, slug } = frontMatter
```

Comment:
nit: Do those need to be exported? They don't seem to be consumed anywhere.

Reply:
I think the mdx to js approach we use expects it to be exported (it was the library complaining about these not being exported), presumably because they do get used somewhere else once the files are compiled. I'm not sure. Here's their docs: https://mdxjs.com/docs/using-mdx/